### DISCUSSION TITLE
QueryParam extract certain Key

### DISCUSSION THREAD

**Discussion by trinadh0030** [mockoon/mockoon#1226] [General]
Hi

```
http://localhost:3000/fapi/v1/batchOrders?batchOrders=[{"side"%3A"BUY"%2C"symbol"%3A"BTCUSDT"%2C"type"%3A"MARKET"%2C"quantity"%3A"0.020000"%2C"reduceOnly"%3A"false"%2C"newClientOrderId"%3A"28a5fbff-7719-4c0e-9d33-0b1fc972b19f"},{"side"%3A"BUY"%2C"symbol"%3A"BTCUSDT"%2C"type"%3A"MARKET"%2C"quantity"%3A"0.020000"%2C"reduceOnly"%3A"false"%2C"newClientOrderId"%3A"532e098f-0335-470b-8842-38f402df65a6"}]&timestamp=1702283341241&signature=5e82cdcbbed63a52c1f6791ef620b11a8a339ff7c4584b22669f770d489214cf
``` 

I am trying to mock the above api. In the response i want to extract the newClientOrderID value and return. Is it possible to extract it

**Comment by 255kb** [comment]
Hi JSON is not supported in the query params. They are parsed using `qs` and [support the bracket notation](https://mockoon.com/docs/latest/api-endpoints/routing/#query-parameters) to create arrays and objects.

**Reply by trinadh0030** [reply]
Hi Guillaume

Is there any plan to support in future. Or is there a way to Json parse the stringified version of json

**Reply by 255kb** [reply]
No there is no such plan. I wonder how common this is. Maybe you can open an issue and see if it gather a bit of interest.

**Reply by trinadh0030** [reply]
https://github.com/mockoon/mockoon/issues/1227#issue-2035361979

**Comment by trinadh0030** [comment]
Hi @255kb 
Is there any support for parsing json object from string

Thanks

**Reply by 255kb** [reply]
Not in the templating helpers (but you opened an issue for this). 
Otherwise, [request's JSON bodies will be parsed](https://mockoon.com/docs/latest/templating/mockoon-request-helpers/#bodyraw) when they have the correct content-type.
